crates/node/polyfills/worker_threads.ts

```diff
@@ -1,20 +1,53 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 // Copyright Joyent and Node contributors. All rights reserved. MIT license.
 
-// TODO(petamoriken): enable prefer-primordials for node polyfills
-// deno-lint-ignore-file prefer-primordials
-
-import { core, internals } from "ext:core/mod.js";
-import { op_require_read_closest_package_json } from "ext:core/ops";
-
-import { isAbsolute, resolve } from "node:path";
+import { core, internals, primordials } from "ext:core/mod.js";
+import {
+  op_create_worker,
+  op_host_post_message,
+  op_host_recv_ctrl,
+  op_host_recv_message,
+  op_host_terminate_worker,
+  op_message_port_recv_message_sync,
+  op_worker_threads_filename,
```

Comment:
You removed `op_worker_threads_filename` in rust land, but in js land, call site still exists. Wouldn't this be a problem?
I think there are a few more cases of this.